.github/workflows/gemini-dispatch.yml

```
@@ -0,0 +1,203 @@
+            🤖 Hi @${{ github.actor }}, I've received your request, and I'm working on it now! You can track my progress [in the logs](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}) for more details.
+          REPOSITORY: '${{ github.repository }}'
+        run: |-
+          gh issue comment "${ISSUE_NUMBER}" \
+            --body "${MESSAGE}" \
+            --repo "${REPOSITORY}"
+
+  review:
+    needs: 'dispatch'
+    if: |-
+      ${{ needs.dispatch.outputs.command == 'review' }}
+    uses: './.github/workflows/gemini-review.yml'
```

actionlint (1.7.11)

[error] could not read reusable workflow file for "./.github/workflows/gemini-review.yml": open /home/jailuser/git/.github/workflows/gemini-review.yml: no such file or directory

(expression)